### ISSUE TITLE
USER ISSUE: not working

### ISSUE THREAD

**Issue by LilpawstheDruid** [StrangeLoopGames/EcoIssues#12450]
**Version:** 0.8.1.4 beta

**Steps to Reproduce:**
will not allow me to join any games at all.
Hi join and connection fails made own server and connection fails.

**Expected behavior:**
hit join and should join server	

**Actual behavior:**
The following error message popped up: connection failed
<size=150%>Connection Failed</size>

Failed to connect - connection timed out.
```

**Do you have mods installed? Does issue happen when no mods are installed?:**
The following error message popped up: connection failed
<size=150%>Connection Failed</size>

Failed to connect - connection timed out.
```

**Comment by LilpawstheDruid** [comment]
okay i finally got it working after much tinkering and stuff. I changed avg setting ect. Finally I broke down and though about an issue me and my husband had with another game where he, while using our game share in steam, could not down load a free dlc for a game I have. I have been trying to play with his copy of eco so i bought it this morning and it finally works.
Thanks for trying to look at it D3nnis3n
